docs/root/configuration/listeners/network_filters/rocketmq_proxy_filter.rst

```diff
@@ -3,6 +3,9 @@
 RocketMQ proxy
 ==============
 
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.network.rocketmq_proxy.v3.RocketmqProxy>`
+* This filter should be configured with the name *envoy.filters.network.rocketmq_proxy*.
```

Comment:
this should be a literal - ie ` ``envoy.filters.network.rocketmq_proxy`` `

Reply:
Done.